### PR TITLE
Update /signup-account request

### PR DESCRIPTION
**Pull request by Kris-MultiSafepay**
- contact_person replaced by user object

Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/account/api-onboarding/create-account.md [hunk 1]
```diff
@@ -105,11 +105,11 @@ curl -X POST "https://testapi.multisafepay.com/v1/json/signup-account" \
       "vat_number": "NL999999999B99",
       "zipcode": "1234 ZP"
     },
-    "contact_person": {
-      "id": 123456,
-      "name": "Contact Person",
-      "password": "***",
-      "title": "mr"
+    "user": {
+      "id": "123456",
```
Comment (Kris-MultiSafepay) [inline content/account/api-onboarding/create-account.md:109]:
```suggestion
```

File: content/account/api-onboarding/create-account.md [hunk 2]
```diff
@@ -105,11 +105,10 @@ curl -X POST "https://testapi.multisafepay.com/v1/json/signup-account" \
...
     },
-    "contact_person": {
-      "id": 123456,
-      "name": "Contact Person",
-      "password": "***",
-      "title": "mr"
+    "user": {
+      "name": "Ad Admin",
+      "email": "admin@funcompany.com",
+      "password": "password"
```
Comment (Kris-MultiSafepay) [inline content/account/api-onboarding/create-account.md:111]:
```suggestion
      "password": "***"
```

**Comment by FrancescaNCook** [comment]
Who do you need to review this one @Kris-MultiSafepay ?

**Comment by Kris-MultiSafepay** [comment]
> Who do you need to review this one @Kris-MultiSafepay ?

I was waiting for a release from backend, which just happened, so I'll merge! Thx :)

**Pull request merged**